hash_practice/exercises.py

```diff
@@ -1,29 +1,164 @@
 
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
-        Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Each subarray will have strings which are anagrams of each other.
+        Time Complexity: O(n^2) -- or is it O(n) bc of the "continue"s?
+        Space Complexity: O(n)
     """

```

Comment:
👍 this works, but can you figure out a way to do this in O(n) time complexity using a dictionary?